pkg/operations/kubernetesupgrade/v162upgrader.go

```diff
@@ -26,7 +26,7 @@ type Kubernetes162upgrader struct {
 // ClusterPreflightCheck does preflight check
 func (ku *Kubernetes162upgrader) ClusterPreflightCheck() error {
 	// Check that current cluster is 1.5.3
-	if ku.DataModel.Properties.OrchestratorProfile.OrchestratorVersion != api.Kubernetes153 {
+	if ku.DataModel.Properties.OrchestratorProfile.OrchestratorVersion != "1.5.3" {
```

Comment:
I just addressed with the latest change. In upgrade, we use OrchestratorRelease through out the code instead of OrchestratorVersion.